src/utils/entitySpec.js

```diff
@@ -180,4 +182,11 @@ export default class EntitySpec {
   toString() {
     return this.toPath()
   }
+
+  static getEntityName(component) {
+    return component.namespace ? `${component.namespace}/${component.name}` : component.name
+  }
+  static getEntityOrigin(component) {
```

Comment:
see comment at the beginning. Feels leaky to expose something about the api URL. Especially here on a relatively generic data structure class.